src/js/controller/controller.js

```diff
@@ -767,7 +767,7 @@ Object.assign(Controller.prototype, {
 
         this.preload = preload;
 
-        /** Controller API / public methods **/
+        /* Controller API / public methods */
```

Comment:
@robwalch will this change expose all methods in the documentation ? Currently only `load` is exposed
![screen shot 2018-10-23 at 11 17 17 am](https://user-images.githubusercontent.com/12138319/47371225-6b27df80-d6b5-11e8-842d-58e0f8aae125.png)

Reply:
Only JSDoc comments should start with /**.

That comment is not a valid JSDoc comment so I removed the extra asterisk, which will remove controller load from the docs output. It will not output any methods.